Makefile

```diff
@@ -29,6 +29,8 @@ CAPI_VERSION?=$(shell grep "sigs.k8s.io/cluster-api" go.mod | awk '{print $$NF}'
 KUBEADM_READY_OS ?= ""
 ELEMENTAL_TOOLKIT_IMAGE ?= ghcr.io/rancher/elemental-toolkit/elemental-cli:nightly
 ELEMENTAL_AGENT_IMAGE ?= ghcr.io/rancher-sandbox/cluster-api-provider-elemental/agent:latest
+ELEMENTAL_OS_IMAGE?=docker.io/local/elemental-capi-os:dev 
+ELEMENTAL_ISO_IMAGE?=docker.io/local/elemental-capi-iso:dev 
```

Comment:
What about systems that don't boot from ISO, like Raspberry Pi ?

Reply:
There's no support yet for disk images. The process of building CAPI images is also manual, I think for this moment best we can do is to add also manual support to invoke `elemental build-disk` as alternative to `build-iso`, but we also need to test this out because we need the disk image to install the system at boot. 

tl;dr Not yet. :P